### ISSUE TITLE
Functionaility not available from keyboard

### ISSUE THREAD

**Issue by xi** [codex-team/editor.js#672]
If I understand correctly, a mouse is required to operate the editor. It therefore violates [WCAG20 2.1](https://www.w3.org/TR/WCAG20/#keyboard-operation).

**Comment by carakas** [comment]
we would like to use this in Fork CMS but this might be a dealbreaker since Fork CMS is certified for accessibility

**Comment by neSpecc** [comment]
Mouse is not required, you can do all manipulations from the keyboard only.

Enter — to create new block.
Shift+Arrows — to select fragment
Tab — to open Toolbox at empty block
Tab — to open Block Settings at non-empty block
Tab (and Shift+Tab) — to switch focused button at the Toolbox, Inline Toolbar or Block Settings panels

and so on.

If we are missed something that you can not do without mouse, please create a separate Issue describing a problem.

**Comment by carakas** [comment]
Ah oke, thank you for the swift reply. Was stuck in a block while tabbing.
